src/CalculatorUITestFramework/NumberPad.cs

```diff
@@ -29,7 +30,7 @@ public class NumberPad
         /// <param name="number">Number to be entered into the calculator.</param>
         public void Input(double number)
         {
-            string numberStr = number.ToString();
+            string numberStr = number.ToString(CultureInfo.CurrentCulture);
```

Comment:
Thank you for this PR, the culture should be "CultureInfo.InvariantCulture", to make sure that we can correctly parse it after

Reply:
Done!